### PR TITLE
Toml update

### PR DESCRIPTION
**Pull request by schandrika**

### CODE REVIEW

File: .github/workflows/run-tests.yml [hunk 1]
```diff
@@ -23,74 +26,61 @@ env:
...
-      uses: snok/install-poetry@v1
-      with:
-        version: 1.2.2
-        virtualenvs-create: true
-        virtualenvs-in-project: true
-        installer-parallel: true
+      - name: Install Poetry
+        uses: snok/install-poetry@v1
+        with:
+          version: 1.2.2
```
Comment (craigpnnl) [inline .github/workflows/run-tests.yml]: Remove the version here

File: pyproject.toml [hunk 2]
```diff
@@ -1,10 +1,6 @@
 [build-system]
 requires = ["poetry-core>=1.0.0"]
```
Comment (craigpnnl) [inline pyproject.toml]: 1.2

File: .github/workflows/make-release.yml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    permissions:
+      contents: write  # To push a branch
+      pull-requests: write  # To create a PR from that branch
+
+    uses: eclipse-volttron/github-tooling/.github/workflows/deploy-release.yml@main
+    with:
+      merge-strategy: ${{ inputs.merge-strategy }}
+      release-version: ${{ inputs.release-version }}
+      bump-rule: ${{ inputs.bump-rule }}
+      run-tests-wait: 600
```
Comment (craigpnnl) [inline .github/workflows/make-release.yml:55]: weren't these going to be not hard coded?

**Review by craigpnnl — CHANGES_REQUESTED** [review]

File: .github/workflows/make-release.yml (hunk 3 above)
Comment (schandrika) [reply .github/workflows/make-release.yml:55]: have to hard code as github action is changing number type inputs into string when passing the value to reusuable workflow and there doesn't seem to be a typecast function available

**Pull request merged**